### ISSUE TITLE
First impressions

### ISSUE THREAD

**Issue by olivierdalang** [qgis/QGIS#35797]
**Describe the bug**

Just reinstalled QGIS from scratch on a new PC, and on first launch, I get this screen :

![image](https://user-images.githubusercontent.com/1894106/79366568-b2aac700-7f4c-11ea-9ed3-e723dbe6178b.png)

I had forgotten about it, but I think we should get rid of it. IMO it looks unprofessional (both the font style and the wording), and doesn't bring much besides slowing down the first launch.

The regular splash screen gives a much better first impression, and especially now that we have the "news" section, it feels redundant.

At the same time, we may want to improve the 

compared to our nice splash screen) 

I'd 

**How to Reproduce**

**QGIS and OS versions**

**Additional context**

**Comment by roya0045** [comment]
I agree that the news panel is oversized and causes confusion to new and existing users when the previous project tab  is hidden under.

**Comment by nyalldawson** [comment]
+1 to removal. It was mostly in place to facilitate upgrades from 2.x, but that's no longer a pressing concern.

**Comment by NathanW2** [comment]
You can remove the Welcome screen if there are no other versions of QGIS installed however if this is a fresh install on a machine with 2.x still on it.  I think we should keep the one that does the settings migrations because there is no doubt people still moving...

I know we have all moved on but other people haven't and I put that there to ease the move

**Comment by alexbruy** [comment]
Also this screen can be useful in future, when another breakage may occur.

**Comment by andreasneumann** [comment]
I agree with Olivier, it looks so ugly to me, and despite the 2.x migration it has zero value. I would be in favor of removing it starting from 3.14. Someone who hasn't yet migrated from 2.x to 3.x will most likely also not migrate to the newest 3.x release, but probably to 3.10.

**Comment by andreasneumann** [comment]
> Also this screen can be useful in future, when another breakage may occur.

Can we keep it and deactivate it?

**Comment by kannes** [comment]
This is long gone and the issue can be closed :)
